pkg/forwarder/backoff_test.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package forwarder
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func TestMinBackoffFactorValid(t *testing.T) {
+	assert.True(t, minBackoffFactor >= 2)
```

Comment:
not sure we are testing anything here ;)

Reply:
I did this to prevent bad values.

`minBackoffFactor = 1` will have no range e.g.:
```
4 4
8 8
16 16 
32 32 
64 64
```

`1 < minBackoffFactor < 2` will have range gaps e.g. `1.6`:
```
2.5 4 
5 8   
10 16
20 32
40 64
```

Reply:
Can you add those comments here then ?
In the future, if we ever change `minBackoffFactor` then people will see the test and edit it too because they wouldn't think of that case